<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QWidget" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gui</string>
  </property>
  <widget class="QLineEdit" name="WordEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PosEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LetterEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="TryButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Try</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PrefEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="WordEdit2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PosEdit2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LetterEdit2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="TryButton2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Try</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PrefEdit2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeButton2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PointsEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PointsEdit2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
